### ISSUE TITLE
Export data file for the graphs

### ISSUE THREAD

**Issue by tomjaguarpaw** [awf/autodiff#25]
It would be nice if the plotting process exported a data file (perhaps JSON) containing the all the datapoints created by the training run.  Ideally the plotting code could be used to replot the graphs from the data file.

**Comment by tomjaguarpaw** [comment]
@mikhailnikolaev @msdmkats are the data files that the graphs are plotted from currently uploaded to the web server?

**Comment by mikhailnikolaev** [comment]
> @mikhailnikolaev @msdmkats are the data files that the graphs are plotted from currently uploaded to the web server?

No, they aren't. Actually, only PNG and HTML files (i.e. static and plotly graphs) are uploaded to the blob container in the current version. Moreover, the plotting process still doesn't collect all time info into the one JSON file.

**Comment by mikhailnikolaev** [comment]
@tomjaguarpaw, I think, we can close this issue due to the PR #183 and #186 (that are parts of #180). What do you think?

**Comment by tomjaguarpaw** [comment]
I would say either

1. Don't close this until JSON export functionality is added to the Azure CI and Batch job, or
2. Close this and create a new issue for the case above
